Sources/BraveWallet/Panels/Add Suggested Token/AddSuggestedTokenView.swift

```diff
@@ -31,7 +31,7 @@ struct AddSuggestedTokenView: View {
         .padding(.top)
         VStack {
           VStack {
-            AssetIconView(token: token, network: cryptoStore.networkStore.selectedChain, length: 64)
+            AssetIconView(token: token, network: cryptoStore.networkStore.defaultSelectedChain, length: 64)
```

Comment:
this is `defaultSelectedChain` not `selectedChainForOrigin`?

Reply:
Hmm, good catch. I believe technically this request could be from any network. I think I need to fix this to `cryptoStore.networkStore.allChains.first(where: {})`.

Reply:
Updated in [084d05c](https://github.com/brave/brave-ios/pull/7470/commits/084d05c615b0b0bf277b3bb1132d0202a73febcc)